honeybee_radiance_folder/folderutil.py

```diff
@@ -0,0 +1,314 @@
+# -*- coding: utf-8 -*-
+"""Utilities for Radiance folder structure."""
+import os
+import json
+
+
+class GeometryState(object):
```

Comment:
Given the ambiguity in the other folder section, I am uncertain if this is non aperture geometry or something else.  `Geometry` just really seems like too vague of a word for what this object actually is.

Reply:
Changed to NonApertureState.